.agents/skills/nixl-cpp-api/references/descriptors-memory.md

```
@@ -0,0 +1,93 @@
+   or storage handles.
+3. Keep the allocation alive until transfers are complete, request handles are
+   released, and the descriptor is deregistered by its owner.
+4. Use backend hints when the operation must be restricted to known backend
+   handles:
+
+```cpp
+nixl_opt_args_t args;
+args.backends.push_back(backend);
+```
+
+5. Register with `registerMem(reg_list, &args)`.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/2/3

(MD029, ol-prefix)